bcda/models/fhir/alr/alr.go

```diff
@@ -1,22 +1,40 @@
 package alr
 
 import (
-	"time"
-
 	"github.com/CMSgov/bcda-app/bcda/models"
-	fhirdatatypes "github.com/google/fhir/go/proto/google/fhir/proto/stu3/datatypes_go_proto"
+	"github.com/CMSgov/bcda-app/log"
 	fhirmodels "github.com/google/fhir/go/proto/google/fhir/proto/stu3/resources_go_proto"
 )
 
+type AlrFhirBulk struct {
+	Patient     *fhirmodels.Patient
+	Coverage    *fhirmodels.Coverage
+	Group       *fhirmodels.Group
+	Risk        []*fhirmodels.RiskAssessment
+	Observation *fhirmodels.Observation
+}
+
 // ToFHIR encodes the models.Alr into a FHIR Patient and N FHIR Observation resources
-func ToFHIR(alr *models.Alr, lastUpdated time.Time) (*fhirmodels.Patient, []*fhirmodels.Observation) {
-	p := patient(alr)
-	p.Meta = &fhirdatatypes.Meta{LastUpdated: fhirInstant(lastUpdated)}
```

Comment:
Why did we drop the last_updated field in the ALR metadata? For non-ALR I beleive we return it with the Bundle resource, should we include it as part of AlrFhirBulk?

Reply:
I don't think I saw it in the implementation guide, but you're right. Let's update this back in another ticket.